conf/messages

```diff
@@ -968,6 +968,13 @@ didYouDisposeOfResidentialProperty.cyaChange=whether UK residential property was
 
 ukResidentCanOnlyReportResidential.title=You cannot use this service
 ukResidentCanOnlyReportResidential.p1=You told us you were a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.agent.p1=You told us your client was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.trust.p1=You told us the trust was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.capacitor.p1=You told us the person was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.personalRep.p1=You told us the person was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
```

Comment:
don't think this should say `on the day **you** sold or disposed of...` - or the capacitor one actually

Reply:
not even sure about the trust one either - but never sure if trust can be called `you` (as the trustee)

Reply:
whoops, I'll take a look again

Reply:
fixed now